<ui version="4.0" >
 <class>GraphPlotter</class>
 <widget class="QDialog" name="GraphPlotter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Graph Plotter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="graphsGroupBox" >
     <property name="title" >
      <string>Graphs to plot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <widget class="QWidget" name="energyPage" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>378</width>
           <height>136</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QCheckBox" name="leftThigh" >
            <property name="text" >
             <string>Left thigh</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="leftLowerLeg" >
            <property name="text" >
             <string>Left lower leg</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rightThigh" >
            <property name="text" >
             <string>Right thigh</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rightLowerLeg" >
            <property name="text" >
             <string>Right lower leg</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="timePage" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>376</width>
           <height>185</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="leftThighAlpha" >
            <property name="text" >
             <string>Left thigh alpha</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="rightThighAlpha" >
            <property name="text" >
             <string>Right thigh alpha</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="leftThighTheta" >
            <property name="text" >
             <string>Left thigh theta</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="rightThighTheta" >
            <property name="text" >
             <string>Right thigh theta</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="leftLowerAlpha" >
            <property name="text" >
             <string>Left lower leg alpha</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QCheckBox" name="rightLowerAlpha" >
            <property name="text" >
             <string>Right lower leg alpha</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QCheckBox" name="leftLowerTheta" >
            <property name="text" >
             <string>Left lower leg theta</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QCheckBox" name="rightLowerTheta" >
            <property name="text" >
             <string>Right lower leg theta</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataSetBox" >
     <property name="title" >
      <string>Data set</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QRadioButton" name="dataSetAuto" >
          <property name="text" >
           <string>Auto (use curve fitting to find 1 period)</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dataSetManual" >
          <property name="text" >
           <string>Manual:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QSpinBox" name="dataSetMax" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>&lt;= t &lt;=</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="dataSetMin" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fftBox" >
     <property name="title" >
      <string>FFT options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QCheckBox" name="magnitude" >
        <property name="text" >
         <string>Magnitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="phase" >
        <property name="text" >
         <string>Phase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="phaseMagnitude" >
        <property name="text" >
         <string>Phase * Magnitude</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QRadioButton" name="displayOnly" >
       <property name="text" >
        <string>Display graphs</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="saveGraphs" >
       <property name="text" >
        <string>Save graphs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="saveData" >
       <property name="text" >
        <string>Save data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="filesGroupBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Destination files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="destDir" />
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="destDirBrowse" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Base filename</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="baseFilename" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>File format</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QComboBox" name="fileFormat" >
        <item>
         <property name="text" >
          <string>PNG</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>JPEG</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Postscript</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>PDF</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>LaTeX</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leftThigh</tabstop>
  <tabstop>leftLowerLeg</tabstop>
  <tabstop>rightThigh</tabstop>
  <tabstop>rightLowerLeg</tabstop>
  <tabstop>destDir</tabstop>
  <tabstop>destDirBrowse</tabstop>
  <tabstop>baseFilename</tabstop>
  <tabstop>fileFormat</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphPlotter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>198</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel" >
     <x>0</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayOnly</sender>
   <signal>toggled(bool)</signal>
   <receiver>filesGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>103</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel" >
     <x>106</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveData</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileFormat</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel" >
     <x>377</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveData</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel" >
     <x>88</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataSetAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>dataSetMax</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>143</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel" >
     <x>109</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataSetAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>223</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>213</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataSetAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>dataSetMin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>276</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>289</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
